cumulus_library/actions/importer.py

```diff
@@ -0,0 +1,81 @@
+import pathlib
+import zipfile
+
+import pandas
+
+from cumulus_library import base_utils, errors
+from cumulus_library.actions import cleaner
+from cumulus_library.template_sql import base_templates
+
+
+def create_table_from_parquet(archive, file, study_name, db, schema):
+    try:
+        parquet_path = pathlib.Path(archive.extract(file))
```

Comment:
Where does this extract to? `pwd`? Feels like we should point it at a temp folder/file.

Reply:
good call, will change.